### PR TITLE
URL token - replace templates + Bug fixed templates

### PR DESCRIPTION
**Pull request by arjun-akto**
12 templates

### CODE REVIEW

File: Broken-Object-Level-Authorization/BOLAURLInsertAPIVersion.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    - v0
+    - v1
+    - v2
+    - v3
+    - v4
+    - v5
+
+execute:
+  type: multiple
+  requests:
```
Comment (ankush-jain-akto) [inline Broken-Object-Level-Authorization/BOLAURLInsertAPIVersion.yml:67]: Whats the vuln here? If I am able to access my own data through v0/v1, it is all OK. Are you missing `replace_auth_header: true` here?

File: Broken-Object-Level-Authorization/BOLAURLReplaceAPIVersion.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          token_replace:
+            location: 1
+            replace_with: ${apiVersions}
+      - success: vulnerable
+      - failure: x2
+    - req:
+      - modify_url:
+          token_replace:
+            location: 2
+            replace_with: ${apiVersions}
```
Comment (ankush-jain-akto) [inline Broken-Object-Level-Authorization/BOLAURLReplaceAPIVersion.yml:82]: Whats the vuln here? If I am able to access my own data through v0/v1, it is all OK. Are you missing replace_auth_header: true here?

**Review by ankush-jain-akto — APPROVED** [review]

File: Broken-Object-Level-Authorization/BOLAURLInsertAPIVersion.yml (hunk 1 above)
Comment (ayushaga14) [reply Broken-Object-Level-Authorization/BOLAURLInsertAPIVersion.yml:67]: need endpoint context check here

Comment (ankush-jain-akto) [reply Broken-Object-Level-Authorization/BOLAURLInsertAPIVersion.yml:67]: `percentage_match > 90`

File: Broken-Object-Level-Authorization/BOLAURLInsertUserID.yml [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            replace_with: ${random_ids}
+      - success: vulnerable
+      - failure: x5
+    - req:
+      - modify_url:
+          token_insert:
+            location: 5
+            replace_with: ${random_ids}
+validate:
+  response_code:
```
Comment (ankush-jain-akto) [inline Broken-Object-Level-Authorization/BOLAURLInsertUserID.yml:102]: check if this endpoint already exists

File: Broken-Object-Level-Authorization/JWTModifyBody.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    - jwt_replace_body: "${jwts}"
+validate:
+  response_code:
+    gte: 200
+    lt: 300
+  response_payload:
+    length:
+      gt: 0
+    percentage_match:
+      lt: 10
```
Comment (ankush-jain-akto) [inline Broken-Object-Level-Authorization/JWTModifyBody.yml:72]: `percentage_match > 90`

**Review by ankush-jain-akto — APPROVED** [review]

**Pull request merged**